mods/cnc/maps/gdi05b/map.yaml

```diff
@@ -0,0 +1,807 @@
+MapFormat: 7
+
+RequiresMod: cnc
+
+Title: Repair the GDI base (b)
+
+Description: A GDI field base is under attack. They have fended off one attack but will not survive another.\n\nMove to the base, repair the structures and then launch a strike force to destroy the Nod base in the area.\n\nDestroy all Nod units and structures.
+
+Author: Westwood Studios
+
+Tileset: TEMPERAT
+
+MapSize: 64,64
+
+Bounds: 8,13,51,47
+
+UseAsShellmap: False
```

Comment:
This has to be changed `Visibility: MissionSelector` to conform with MapFormat 7.